recorder/src/main/resources/tracking.js

```diff
@@ -73,6 +73,7 @@ jsflight.TrackHash = function(event) {
         console.log(e);
     } finally {
         jsflight.eventId++;
+        jsflight.rebindAllHandlers();
```

Comment:
yep. you shall use both ways - use Observer and it isn't supported fall back to MutationEvents
http://stackoverflow.com/a/14570614/911397

``setInterval(..)`` is the most unpredictable way, imho